compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -2348,7 +2348,7 @@ class Typer extends Namer
       def dummyArg(tp: Type) = untpd.Ident(nme.???).withTypeUnchecked(tp)
 
       def addImplicitArgs(implicit ctx: Context) = {
-        def implicitArgs(formals: List[Type]): List[Tree] = formals match {
+        def implicitArgs(formals: List[Type], n: Int): List[Tree] = formals match {
```

Comment:
either give a more explicit name for this parameter or add some documentation for this method (something that would be nice to have anyway :))

Reply:
Renamed to `argIndex`.